### DISCUSSION TITLE
Tilt correction in MTEX post-processing

### DISCUSSION THREAD

**Discussion by Remsdel** [mtex-toolbox/mtex#783] [Ask Anything]
We have a set of EDAX EBSD maps for which the tilt correction has been improperly calibrated. It results in a grain elongation in the Y-direction (vertical). Apparently it is not possible to correct this improper calibration a posteriori from the EDAX software on the already acquired images. We do have corresponding BSE images with grain boundaries of the same areas from which one can determine the stretch in the Y-direction. So in principle a correction factor for the Y direction can be determined. We were wondering is MTEX would offer a way to correct for it during post-processing of the data. Thanks for your feedback.

**Comment by kilir** [comment]
Hi,
yes, you can set x,y-coordinates (ebsd.x, ebsd.y) anytime.
Cheers,
Rüdiger
